### PR TITLE
FW-1838 - Create new endpoints for simple tasks

### PR DESCRIPTION
**Pull request by dy2288**
See postman/ticket for documentation.

### CODE REVIEW

**Comment by dy2288** [comment]
Note: Should have opened as a draft. Still need to create tests for end-points.

File: .gitattributes [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
 * text=auto
 
+# Ensure .sh files are always encoded as LF
```
Comment (dy2288) [inline .gitattributes:4]: These files seem to be converted sometimes to CRLF, and not handled correctly despite `core.autocrlf` set appropriately. Would be best to ensure they are always converted to LF line endings.

File: FirstVoicesRESTPageProviders/UpdateModule.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (dy2288) [inline FirstVoicesRESTPageProviders/UpdateModule.sh:1]: RESTPageProviders was missing UpdateModule.sh

File: FirstVoicesSecurity/src/main/java/ca/firstvoices/securitypolicies/groups/LanguageRecorders.java [hunk 3]
```diff
@@ -54,23 +54,21 @@
...
    */
-  private Boolean hasPermissionInACP(ACP mergedAcp, List<String> additionalPrincipalsList,
-      String permission) {
+  private boolean hasRecordPermissionInACP(ACP mergedAcp, List<String> additionalPrincipalsList) {
 
     for (ACL acl : mergedAcp.getACLs()) {
-      for (ACE ace : acl.getACEs()) {
-        if (ace.isGranted() && additionalPrincipalsList.contains(ace.getUsername()) && ace
-            .getPermission().equals(permission)) {
+      for (ACE ace : acl.getACEs()) { // get permission CAN be null
```
Comment (dy2288) [inline FirstVoicesSecurity/src/main/java/ca/firstvoices/securitypolicies/groups/LanguageRecorders.java:69]: Found a bug here as the Nuxeo task service will set some custom permissions on tasks, where the permission itself can be null. Rest of the changes here are SonarQube.

**Review by elysse-fpcc — APPROVED** [review]
The changes seem good to me. A couple things I was confused about but I think I answered all my questions after reading through everything. Approved 😸

**Pull request merged**